module/test.c

```diff
@@ -39,12 +39,27 @@ int main(void)
   void * mem = mmap(NULL, create.size, PROT_READ | PROT_WRITE, MAP_SHARED, dmaFd, 0);
   if (!mem)
   {
-    perror("mmap");
+    perror("mmap on dmabuf");
     return -1;
   }
 
   memset(mem, 0xAA, create.size);
+
+  int page_size = getpagesize();
```

Comment:
Just comment what this test does exactly please, at a glance, it looks like left-behind debug code :)

Reply:
Going to move this test to #457.